src/accessibility/images/index.md.njk

```diff
@@ -0,0 +1,16 @@
+---
+title: Images
+description: Images
+section: Accessibility
+theme: Resources
+layout: layout-pane.njk
+order: 8
+---
+
+Not everyone reading your document will be able to see an image or screenshot so you must always add 'alternative text' or 'alt text'. 
+
+This means people using screen readers can read or hear a description of the image. 
+
+If the image is purely decorative then you can mark it as 'decorative only'. But if this is the case, you might want to think about whether you need to include the image at all. [Images, Describing images, Graphs and charts, Diagrams, Infographics, Image copyright standard](https://www.gov.uk/guidance/content-design/images).  
```

Comment:
Might be worth mentioning that for decorative images you should still declare a blank `alt` attribute: `alt=""`

https://www.w3.org/TR/WCAG20-TECHS/H67.html#H67-description